xbmc/cores/AudioEngine/CMakeLists.txt

```diff
@@ -131,6 +134,9 @@ endif()
 
 core_add_library(audioengine)
 target_include_directories(${CORE_LIBRARY} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
+if(OVERRIDES_CPP)
+  target_compile_definitions(${CORE_LIBRARY} PRIVATE -DPLATFORM_OVERRIDE)
```

Comment:
Thx for the feedback I added some stuff. Btw. aren't the headers missing here: https://github.com/fetzerch/xbmc/blob/8108034415de1882e127ec64d0de9918cf236d2e/xbmc/cores/VideoPlayer/Process/Makefile.in#L9